### ISSUE TITLE
automate releasing to play store and github

### ISSUE THREAD

**Issue by sds100** [sds100/KeyMapper#576]
- do before starting 2.3 beta releases
- use fastlane to automate creating releases on GitHub and Google Play

- [x] run unit tests and instrumentation tests before publishing. send message to discord if failed. DO NOT RUN THIS IN A SEPARATE TASK TO ASSEMBLING THE APK

**Comment by sds100** [comment]
implemented
